### PR TITLE
fix infinite lane scroll

### PR DESCRIPTION
**Pull request by gevhambarzumyan93**
naeq ara

### CODE REVIEW

**Review by oliviervanbulck — APPROVED** [review]

**Comment by orlandovallejos** [comment]
I applied a fix following the suggestion another user made. I don't remember who, but he did something like this: https://github.com/workast/react-trello/commit/b35afdb6d2df5c3670ec81b0afd628d06d8ef9a0

The idea is to expose a method where the user can decide if a Lane should paginate or not. In my case, I only want one of them to paginate, so using this method I can return or just call the onScroll method.

**Comment by gevhambarzumyan93** [comment]
> I applied a fix following the suggestion another user made. I don't remember who, but he did something like this: [workast@b35afdb](https://github.com/workast/react-trello/commit/b35afdb6d2df5c3670ec81b0afd628d06d8ef9a0)
> 
> The idea is to expose a method where the user can decide if a Lane should paginate or not. In my case, I only want one of them to paginate, so using this method I can return or just call the onScroll method.

I just can't find this line necessary if (!moreCards || moreCards.length === 0)  because, if there are no more cards then why try load more

**Comment by dapi** [comment]
Hi guys! What is the better way to reproduce this bug and test fixes? Could we write a tests for it?

**Comment by gevhambarzumyan93** [comment]
> Hi guys! What is the better way to reproduce this bug and test fixes? Could we write a tests for it?

just scroll down any of the lanes. when there is nothing to get from db it jumps up, so you can't see the last task in the lane

File: src/controllers/Lane.js [hunk 1]
```diff
@@ -31,10 +31,7 @@ class Lane extends Component {
       this.setState({loading: true})
       const nextPage = currentPage + 1
       onLaneScroll(nextPage, this.props.id).then(moreCards => {
-        if (!moreCards || moreCards.length === 0) {
-          // if no cards present, stop retrying until user action
-          node.scrollTop = node.scrollTop - 100
```
Comment (dapi) [inline src/controllers/Lane.js:36]: Why did you remove this line? It has beed added to fix this issue - https://github.com/rcdexta/react-trello/issues/54

Comment (gevhambarzumyan93) [reply src/controllers/Lane.js:36]: it's is the reason for the problem, 
when there is no card, so on the bottom of the lane your scroll jumps up by 100, and the last card stays covered. so you need to scroll down very slowly or make your cards bigger.
so this line did not fix anything.
 if there isn't any card why make scroll move?

Comment (oliviervanbulck) [reply src/controllers/Lane.js:36]: @dapi I second what @gevhambarzumyan93 says. It's the whole reason why this is happening. I don't know why it was added back then but at least right now it doesn't seem to have any use anymore. I tested his changes locally too and it's all working smooth as it should be.

Comment (dapi) [reply src/controllers/Lane.js:36]: Guys, thank you!
